filament/backend/src/opengl/OpenGLDriver.h

```diff
@@ -249,7 +249,8 @@ class OpenGLDriver final : public backend::DriverBase {
 
     // Memory management...
 
-    backend::HandleAllocator mHandleAllocator;
+    // See also the explicit template instantiation in HandleAllocator.cpp
+    backend::HandleAllocator<16, 64, 208> mHandleAllocator;
```

Comment:
You could define an alias inside the `#if FILAMENT_SUPPORTS_FOO`, for e.g.
`using HandleAllocatorGL = HandleAllocator<16,64,208>;`
So that we don't have to repeat the sizes here?

Reply:
Yeah but the `#if FILAMENT_SUPPORTS_FOO` thingie is in a cpp file, and C++ doesn't let you instantiate a template using an alias, as far as I can tell.

Reply:
Maybe I could do this with a macro...

Reply:
Done.